charts/nr-k8s-otel-collector/values.yaml

```diff
@@ -8,12 +8,15 @@ kube-state-metrics:
   # metrics by default. You can enable the target labels/annotations metrics to be monitored by using the metricLabelsAllowlist/metricAnnotationsAllowList options described [here](https://github.com/prometheus-community/helm-charts/blob/159cd8e4fb89b8b107dcc100287504bb91bf30e0/charts/kube-state-metrics/values.yaml#L274) in
   # your Kubernetes clusters.
   enabled: true
-  prometheusScrape: false  # Disable prometheus from auto-discovering KSM and potentially scraping duplicate data.
+  # -- Disable prometheus from auto-discovering KSM and potentially scraping duplicated data
+  prometheusScrape: false  
 
 image:
+  # -- OTel collector image to be deployed. NRDOT is the default value but you can use your own collector as long it accomplish the following requirements 
```

Comment:
Should we also add what the following requirements are?

Reply:
I agree, that would be helpful